IBPSA/Fluid/Chillers/ModularReversible/Data/TableData2D/Generic.mo

```diff
@@ -0,0 +1,37 @@
+within IBPSA.Fluid.Chillers.ModularReversible.Data.TableData2D;
+record Generic "Basic chiller data"
+  extends IBPSA.Fluid.HeatPumps.ModularReversible.Data.TableData2D.Generic;
+
+  parameter Real tabQEva_flow[:,:]
+    "Cooling power consumption table, T in degC, Q_flow in W";
```

Comment:
Let's not use degC. Use K, otherwise it will be a disaster as users for sure mix up units.

Reply:
The values are already in K, I just forgot to change the description string.